resources/js/components/Menu.vue

```diff
@@ -125,6 +133,7 @@ export default {
       newItems: this.initialNewItems,
       sectionRight: true,
       items: [],
+      selectedLanguage: "es",
```

Comment:
Is this correct Maybe we should have en as default?

Reply:
This part will be changed when the Flag button will be used. This part is just for testing purposes.